README.md

```diff
@@ -831,15 +838,19 @@ $ jest '<path to test>'
 ## Open source @ IBM
 [Find more open source projects on the IBM Github Page.](http://ibm.github.io/)
 
-## License
+## Contributing
+See [CONTRIBUTING](https://github.com/watson-developer-cloud/node-sdk/blob/master/.github/CONTRIBUTING.md).
+
+## Featured Projects
+We love to highlight cool open-source projects that use this SDK! If you'd like to get your project added to the list, feel free to make an issue linking us to it.
+- [Watson Speech to Text Demo App](https://github.com/watson-developer-cloud/speech-to-text-nodejs)
+- [Watson Assistant Demo App](https://github.com/watson-developer-cloud/assistant-simple)
```

Comment:
Let's use `assistant-demo` not `assistant-simple`

Reply:
can you search on github for projects using `watson-developer-cloud`. sort them by stars